feat(webpack-constants): add constants package (#1114)
electron-userland/electron-forge: packages/utils/webpack-constants/README.md

# @electron-forge/webpack-constants

> > Expose electron forges webpack magic constants

Reviewer: ```suggestion
> Expose Electron Forge's magic constants for Webpack
```

> This module makes it simple and type safe to use the webpack magic constants

Reviewer: ```suggestion
This module provides a type-safe interface to use the Webpack plugin's magic constants for referencing Webpack-generated bundles, as
```

> provided by @electron-forge.

Reviewer: ```suggestion
defined by Electron Forge.
```

## Usage

> ```ts

Reviewer: ```suggestion
```typescript
```
